osbenchmark/benchmark.py

```diff
@@ -1203,6 +1216,10 @@ def main():
     # Early init of console output so we start to show everything consistently.
     console.init(quiet=False)
 
+    # Handle command suggestions before argument parsing
+    if handle_command_suggestions():
+        return
```

Comment:
While this works, `sys.exit(1)` is more suitable to clearly communicate to users and developers that this error prevents the tool from operating.